Gemfile

```diff
@@ -14,6 +14,7 @@ group :development, :test do
   gem 'coveralls', require: false
   gem 'rubocop', '~> 0.37.2', require: false
   gem 'rubocop-rspec', require: false
+  gem 'rake', '< 11.0'
```

Comment:
@jcoyne I think it has to do with the ruby version.  You can see the error in travis: https://travis-ci.org/projecthydra/sufia/jobs/198929354